vapoursynth/src/api.rs

```diff
@@ -563,10 +361,10 @@ impl API {
     #[inline]
     pub(crate) unsafe fn get_frame_write_ptr(
         self,
-        frame: &mut ffi::VSFrameRef,
+        frame: &mut ffi::VSFrame,
         plane: i32,
     ) -> *mut u8 {
-        (self.handle.as_ref().getWritePtr)(frame, plane)
+        (self.handle.as_ref().getWritePtr.unwrap())(frame, plane)
```

Comment:
All these unwraps are unfortunate due to bringing the whole panic path to all these functions. But they require postprocessing bindgen bindings to deal with... Maybe generate-bindings.sh could somehow automatically remove those Options where they shouldn't be?